src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -52,11 +55,22 @@ public void Abort()
          {
             View.ValidationIsRunning(true);
             await _batchStarterTask.StartBatch(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token);
+            await _batchComparisonTask.StartComparison(_outputFolderDTO.FolderPath, _cancellationTokenSource.Token);
          }
```

Comment:
This needs a catch to OperationCanceledException (even thought StartBatch does not raise that at the moment). This is an implementation detail but in general, _cancellation is always done with the exception handling

Maybe it would be good to have the StartBAtch throw the exception so that a message could be shown to user on log that it was canceled